source/customizing-application/editor-ui/creating-custom-editor.rst

```diff
@@ -1,911 +1 @@
-.. _creating-custom-editor:

Customizing editor
##################


.. _creating-custom-editor-editor:

-        
-A metadata editor configuration is defined for a specific schema plugin standard
```

Comment:
Better to fix formatting issue in here https://github.com/geonetwork/core-geonetwork/blob/master/schemas/config-editor.xsd#L30 as this doc page is created from it. See https://geonetwork-opensource.org/manuals/trunk/en/contributing/writing-documentation.html#building-the-standards-docs